### DISCUSSION TITLE
How To: Only Tiles, no OSM

### DISCUSSION THREAD

**Discussion by anthraxn8b** [Ortho4XP/Ortho4XP#58] [Q&A] [answered]
Is it possible to instrument Ortho4XP in a way that it only generates the tiles and leaves out everything that World2XP generates?
Reason: I get better results regarding the OSM part using W2XP.

**Comment by d41k4n** [comment]
I don't understand your question. Ortho4XP has no overlap at all with W2XP since it generates a "clean" base mesh DSF without any overlays. W2XP by contrast only deals with overlays. If you "only" want to generate "tiles" without overlays, then that's what you already get by default, unless you extract existing overlays as yOrtho4XP_Overlays and put that on top of the final ortho scenery in scenery_packs.ini.

**Reply by anthraxn8b** [reply]
Maybe then I have an incorrect understanding of how Ortho4XP works.

If I get it correctly "yOrtho4XP_Overlays" is only the byproduct - **REQUIRED BY ITS MAIN JOB(!?)** - that can be used but normally should not.

Correct?

**Comment by d41k4n** [comment, marked as answer]
Extracting overlays is an optional step in Ortho4XP's pipeline. The reason it exists is because if you only have global default scenery installed then adding an ortho base mesh DSF will suppress all default overlays for that DSF tile. A typical case where this isn't required is with the Central Europe region when [Simheaven's X-World Europe](https://simheaven.com/simdownloads/x-world-europe/) scenery is installed which excludes and replaces all global default overlays anyway.

**Reply by anthraxn8b** [reply]
@d41k4n THX!
